config/http.conf

```diff
@@ -2,12 +2,19 @@ http {
   include /etc/nginx/mime.types;
   include /etc/nginx/log.conf;
 
-  server_tokens         off;
-  keepalive_timeout     20s;
-  sendfile              on;
-  tcp_nopush            on;
-  client_max_body_size  400m;
-  client_body_timeout   300s;
+  server_tokens           off;
+  sendfile                on;
+  tcp_nopush              on;
+
+  client_max_body_size    500m;
```

Comment:
Any reason for increasing to `500m`?

Reply:
Not real reason other than it feels like a nice round number 😅 ... `400m` feels very random.